mkt/settings.py

```diff
@@ -204,7 +205,11 @@ def path(*args):
     # a middleware causes a redirect, then other middleware stops processing.
     'mkt.site.middleware.RemoveSlashMiddleware',
     'mkt.site.middleware.CommonMiddleware',
+    'djangosecure.middleware.SecurityMiddleware',
 )
+# prevent the browser from guessing the content type,
+# and force it to always use the type provided in the Content-Type header
```

Comment:
Comments should be a full sentence that starts with a capital letter and ends with a period.